src/main/java/frc/robot/Limelight.java

```diff
@@ -1,17 +1,57 @@
+package frc.robot;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Transform2d;
+import edu.wpi.first.math.geometry.Translation2d;
 import edu.wpi.first.networktables.NetworkTable;
-import edu.wpi.first.networktables.NetworkTableEntry;
 import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.util.sendable.SendableBuilder;
+import static frc.robot.Constants.LimelightConstants.*;
+
 
 class Limelight {
 
+    /**
+     * Creates a network table instance and an entry for each necessary value
+     */
```

Comment:
change this comment: instantiating the `NetworkTable` object in code neither creates a network table instance nor creates entries for values; it only provides an object through which to access the existing entries